### PR TITLE
Implement option serialize and deserialize

### PR DESCRIPTION
**Pull request by duanyytop**

### CODE REVIEW

File: src/option/index.ts [hunk 1]
```diff
@@ -1 +1,32 @@
-export default {}
+import { assertIsHexStr } from '../utils'
+import { deserializeArray } from '../array'
+
+export const serializeOption = (origin: TypeElement[]) => {
```
Comment (Keith-CY) [inline src/option/index.ts]: Why is the type of `option`'s argument `[string, any]`?

Comment (duanyytop) [reply src/option/index.ts]: No. `declare type TypeElement = [string, string]` not `[string, any]`

Comment (Keith-CY) [reply src/option/index.ts]: According to the doc, the `option` has zero or one element, so what does the first string do here?

Comment (duanyytop) [reply src/option/index.ts]: Updated. https://github.com/Keith-CY/molecule-javascript/pull/12/commits/2bfe30878dc633b5ad6c65dfb5d74d425c3b6604

**Pull request merged**